hist/hist/inc/THn.h

```diff
@@ -245,7 +245,6 @@ typedef THnT<Double_t> THnD;
 typedef THnT<Char_t>   THnC;
 typedef THnT<Short_t>  THnS;
 typedef THnT<Int_t>    THnI;
-typedef THnT<Long_t>   THnL;
-typedef THnT<Long64_t> THnL64;
```

Comment:
I can agree in principle with the suggestion, but removing a public typedef will break user code, I don't know if we can so simply remove it. Maybe leaving `THnL64` and then aliasing `THnL` to the same type would be the safer approach. @lmoneta what do you think?

Reply:
Yep, I agree... that's why I preferred to just call it TH1L64 or TH1LL, it avoids ambiguity and domino side-effects on this nightmare PR. IMHO, a new incremental feature suggestion should not be charged upon (or blocked by) fixing bad old year-long design conventions.

Anyway, so should I then do `typedef THnT<Long64_t>   THnL; typedef THnT<Long64_t>   THnL64;` and adapt the roottest PR?

Reply:
I acknowledge this PR went way beyond the necessary changes, but I believe we are very close to merging. My only remaining request would be to avoid removing already existing public interfaces (i.e. their symbols) just to avoid breaking existing user code. Then this should be merged finally. Sorry again for such a long wait

Reply:
Thanks, I think I have fixed it now. Cheers!